gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -373,7 +374,9 @@ class GateConfig extends RedisHttpSessionConfiguration {
    */
   @Bean
   FilterRegistrationBean authenticatedRequestFilter() {
-    def frb = new FilterRegistrationBean(new AuthenticatedRequestFilter(false, true, true))
+    // no need to force the `AuthenticatedRequestFilter` to create a request id as that is
+    // handled by the `RequestTimingFilter`.
```

Comment:
If RequestTimingFilter is turned off, wouldn't we miss request ID in MDC ?

Reply:
Mmm, good point. It probably does make sense to have the other filter on by default across-the-board.